src/components/navigation/header.tsx

```diff
@@ -1,29 +1,35 @@
+// import dynamic from 'next/dynamic'
 import Router from 'next/router'
 import React from 'react'
 import styled from 'styled-components'
 
-import { menuData } from '../../data/menu'
-import { makeResponsivePadding } from '../../helper/css'
 import { Logo } from './logo'
 import { Menu } from './menu'
 import { MobileMenu } from './mobile-menu'
 import { MobileMenuButton } from './mobile-menu-button'
 import { SearchInput } from './search-input'
+import { useAuth } from '@/auth/use-auth'
+import { getMenuData } from '@/data/menu'
+import { makeResponsivePadding } from '@/helper/css'
 
 export function Header() {
   const [isOpen, setOpen] = React.useState(false)
+  const auth = useAuth()
+  const links = getMenuData()
+
   Router.events.on('routeChangeStart', () => {
     setOpen(false)
   })
+
   return (
     <BlueHeader>
       <MobileMenuButton onClick={() => setOpen(!isOpen)} open={isOpen} />
       <PaddedDiv>
-        <Menu links={menuData}></Menu>
+        <Menu links={links} auth={auth} />
         <Logo subline="Die freie Lernplattform" />
       </PaddedDiv>
       <SearchInput />
-      {isOpen && <MobileMenu links={menuData} />}
+      {isOpen && <MobileMenu links={links} />}
```

Comment:
Mobile menu is missing auth stuff.

Reply:
right, will there be useful functions for logged in mobile users? 
commenting I guess.

Reply:
Yes. With the new editor it should also be possible to make small edits on mobile. And any learner features that we introduce in the future.

Reply:
My local auth setup is broken again, so I can't test thoroughly. 
But MobileMenu should now have the same functionality. It's a bit easier because we render the menu only when the user open it.